### PR TITLE
[CHE 6] Fix Organization selenium tests

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
- Fix **UserOrganizationTest**, **RenameOrganizationTest** and **OrganizationTest** selenium tests.

### CODE REVIEW

File: selenium/che-selenium-test/src/test/resources/suites/OrganizationSuite.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!--
```
Comment (tolusha) [inline selenium/che-selenium-test/src/test/resources/suites/OrganizationSuite.xml]: Why do we need a new suite?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AdminOrganizationTest.java [hunk 2]
```diff
@@ -51,19 +50,16 @@
...
   @Inject private NavigationBar navigationBar;
   @Inject private CheMultiuserAdminDashboard dashboard;
+  @Inject private TestUser testUser;
 
   @Inject
   @Named("admin")
   private TestOrganizationServiceClient testOrganizationServiceClient;
 
-  @Inject private TestUser testUser;
-  @Inject private AdminTestUser adminTestUser;
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AdminOrganizationTest.java:65]: Looks strange - **AdminOrganizationTest** without **AdminTestUser**. It seems it doesn't test Admin dashboard at all. Do you have an issue to fix this moment?

**Review by dmytro-ndp — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AdminOrganizationTest.java (hunk 2 above)
Comment (SkorikSergey) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AdminOrganizationTest.java:65]: Tests from organization package will be reworked by https://github.com/eclipse/che/issues/7121.

File: selenium/che-selenium-test/src/test/resources/suites/OrganizationSuite.xml (hunk 1 above)
Comment (SkorikSergey) [reply selenium/che-selenium-test/src/test/resources/suites/OrganizationSuite.xml]: This suite need for starting tests only from organization package.

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/resources/suites/OrganizationSuite.xml]: You can try to run all tests from package with next command where two asterisks are used, as @tolusha suggested earlier: 
```
che-selenium-test/selenium-tests.sh --test=org.eclipse.che.selenium.dashboard.organization.**
```

**Review by dmytro-ndp — APPROVED** [review]

File: selenium/che-selenium-test/src/test/resources/suites/OrganizationSuite.xml (hunk 1 above)
Comment (tolusha) [reply selenium/che-selenium-test/src/test/resources/suites/OrganizationSuite.xml]: Remove suite if the command above works

**Review by tolusha — APPROVED** [review]

**Pull request merged**